docs/presentations/linkedin-meetup-2024-04-24.qmd

```diff
@@ -0,0 +1,543 @@
+---
+title: "Ibis"
+subtitle: "Portable Python DataFrames"
+author:
+  - Chloe He
+  - Phillip Cloud
+format:
+  revealjs:
+    smaller: true
+    footer: <https://ibis-project.org>
+    preview-links: true
+    # https://quarto.org/docs/presentations/revealjs/themes.html#using-themes
+    theme: night
+---
+
+# Who
+
+:::: {.columns}
+
+::: {.column width="50%"}
+Phillip Cloud
+
+- Principal engineer at Voltron Data
+- Python analytics for 10+ years
+- Open source
+- Tech lead for Ibis
+:::
+
+::: {.column width="50%"}
+Chloe He
+
+- …
+- …
+- …
+:::
+
+::::
+
+# What
+
+## Ibis is a Python frontend for:
+
+- Exploratory data analysis
+- General analytics
+- Data engineering
+- ML preprocessing
+- `$OTHER_THINGS`
+
+# Examples
+
+## Examples
+
+<https://ibis-project.org/tutorials/getting_started>
+
+::: {.panel-tabset}
+
+## Raw data
+
+```{python}
+#| echo: true
+from ibis.interactive import *
+
+penguins = ibis.examples.penguins.fetch()
+penguins
+```
+
+## Aggregate penguins
+
+```{python}
+#| echo: true
+(
+  penguins.group_by("species", "island")
+  .agg(
+    n=_.count(),
+    avg_bill_length=_.bill_length_mm.mean(),
+    avg_flipper_length=_.flipper_length_mm.median()
+  )
+)
+```
+
+## Normalize numerics
+
+```{python}
+#| echo: true
+expr = (
+  penguins.group_by("species")
+  .mutate(s.across(s.numeric() & ~s.c("year"), (_ - _.mean()) / _.std()))
+  .head(5)
+)
+expr
+```
+
+:::
+
+# Interlude
+
+## SQL
+
+:::: {.r-stack}
+::: {.fragment}
+
+```{python}
+#| echo: true
+expr = (
+  penguins.group_by("species")
+  .mutate(s.across(s.numeric() & ~s.c("year"), (_ - _.mean()) / _.std()))
+  .head(5)
+)
+```
+
+::: {.fragment .fade-in-then-semi-out}
+```{python}
+#| echo: true
+ibis.to_sql(expr)
```

Comment:
It looks like the output here is going off the slide?

Reply:
Yeah, that's intentional for now.